### ISSUE TITLE
Confusing plugin insertion on Ubuntu

### ISSUE THREAD

**Issue by Sepero** [kee-org/KeeFox#546]
Since leaving LastPass, I've been happily using Keefox for over a year now. After battling to put in the KeePass plugin manually about once or twice, I wrote a tiny script to automate it for me.

The battle is Not putting in the plugin. Actually putting in the plugin is very easy. The battle is figuring out Where to put the plugin, because the instructions are a tad misleading.

My script is basically just one line:

```
cat "$HOME/.mozilla/firefox/1ofloojf.default/extensions/keefox@chris.tomlinson/deps/KeePassRPC.plgx" | sudo tee /usr/lib/keepass2/deps/KeePassRPC.plgx > /dev/null
```

The key location being `/usr/lib/keepass2/deps/`.

I would like to request additions to quick help pages such as this: chrome://keefox/content/install_mono.xul?upgrade=1
1. Add a hint to the location `/usr/lib/keepass2/deps/`
2. Add a note that KeePass will need to be restarted if it is already running.

Best Regards

**Comment by dlech** [comment]
This is why I maintain a KeePass plugin ppa. See <https://github.com/luckyrat/KeeFox/wiki/en-|-Installation-|-Linux#install-the-firefox-extension>.

Still, updating that page would be beneficial for those that don't use the ppa. However, recent versions of KeeFox expect to find the .plgx in `/usr/lib/keepass2/plugins/`, not `/usr/lib/keepass2/deps/`. Also, your script should include `sudo mkdir -p /usr/lib/keepass2/plugins` to create the `plugins` directory in case it does not exist (if you want it to work for anyone that has not installed KeeFox before).

**Comment by Sepero** [comment]
> However, recent versions of KeeFox expect to find the .plgx in `/usr/lib/keepass2/plugins/`, not `/usr/lib/keepass2/deps/`.

I did not know about this. Perhaps deprecated? I just tested using plugins directory, and yes this new directory works for me too, so thank you for updating me. And good idea adding in the `mkdir`. :)

I'm on Ubuntu 14.04 LTS.

I don't know if there are any other "quick help" or "first time setup" pages, but I feel it would be good to add the info to any of those pages. Cheers
